pyccel/ast/variable.py

```diff
@@ -817,13 +817,12 @@ def __getitem__(self, *args):
         j = 0
         for i in self.indices:
             if isinstance(i, Slice) and j<len(args):
-                # TODO: Replace with simplify = True in PR 797
                 if j == 0:
                     i = args[j]
                 elif i.step == 1:
-                    i = PyccelAdd(i.start, args[j])
+                    i = PyccelAdd(i.start, args[j], simplify = True)
                 else:
-                    i = PyccelAdd(i.start, PyccelMul(i.step, args[j]))
+                    i = PyccelAdd(i.start, PyccelMul(i.step, args[j]), simplify = True)
```

Comment:
Do we also have a `simplify = True` for `PyccelMul` to handle `1*x`?

Reply:
Should I do the same for `PyccelDiv`

Reply:
Why not. I don't think it's used as much but it can't do any harm :+1: